### ISSUE TITLE
Projects Files/Links Pass Through

### ISSUE THREAD

**Issue by acshields** [topcoder-platform/work-manager#901]
Explore whether there is a way to pass-through files and links that have been shared in project to Work Manager, so that they display in an "available attachments" folder when the user seeks to attach a file or link to the challenge or task.

This feature could save the person who is managing the delivery the effort of having to download a file from the project, then reshare it in the challenge.

**Comment by vikasrohit** [comment]
@acshields I guess this should be possible however, my concern is that we would end up replicating almost every thing from projects app to the work manager if we go with this approach because IMO, some day we would need to show discussions, links and scope from the projects app. This is still my biggest concern with separating work manager from projects app and micro framing apps. We would end up more code replication with such approaches and things would go more inconsistent over the time. But if we have good trade offs to offset these negatives, we can go ahead with this approach.

**Comment by acshields** [comment]
@vikasrohit - if we added a way to tag an uploaded file/link as something that will be needed for a challenge, could that cut down on the replicated info we need to manage?

Also as a note, these are the things I can see us needing to "carry over" from project in the future (not fully applicable to this specific issue, but sharing in response to your thoughts above):
-Files & Links that will need to shared on the challenge/task
-Specs that the client/delivery team collaborate on for a challenge/task (in the short term this would probably just be a file, where the Copilot copy/pastes text, but in the long term I have an idea for an in-app spec review, using a not yet created "review app")
-The project's billing account
-NDA preferences that are set at the project level and are passed through to all challenges/tasks
-Scorecard preferences that are set at the project level and are passed through to all challenges/tasks

**Comment by vikasrohit** [comment]
> if we added a way to tag an uploaded file/link as something that will be needed for a challenge, could that cut down on the replicated info we need to manage?

No, it is not the data which we are replicating but it is the UI that is being replicated.

Thanks for the other information that we might want to carry over from projects to work manager. However, don't you think it now covers pretty much every information that we capture in projects right now? So, essentially we would end up in replicating a view (UI) for every thing that we show in projects app.
